scripts/lib/linkcheck/checks/good-link-label.ts

```diff
@@ -0,0 +1,38 @@
+import { DomUtils } from 'htmlparser2';
+import kleur from 'kleur';
+import { dedentMd } from '../../output.mjs';
+import { CheckBase, CheckHtmlPageContext } from '../base/check';
+import { IssueType } from '../base/issue';
+
+/** List of labels that are insufficiently descriptive for a link. */
+const blocklist = new Set(['read more', 'here', 'more']);
```

Comment:
**suggestion:** Add `click here` as an option? That's a pretty trope-y generic link text.

Reply:
Great suggestion! Thanks for taking the time to review @BenDMyers 💜